dev/docker/Dockerfile.cpu_and_deepspeed

```diff
@@ -29,8 +29,8 @@ COPY ./MANIFEST.in .
 
 RUN mkdir ./finetune && mkdir ./inference
 
-RUN --mount=type=cache,target=/root/.cache/pip pip install -e .[cpu,deepspeed] -f https://developer.intel.com/ipex-whl-stable-cpu \
-    -f https://download.pytorch.org/whl/torch_stable.html
+RUN --mount=type=cache,target=/root/.cache/pip pip install -e .[cpu,deepspeed] --extra-index-url https://download.pytorch.org/whl/cpu \
+    --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/cpu/us/
```

Comment:
Why some use "--index-url  --extra-index-url", some use "--extra-index-url --extra-index-url"? Is there any difference? But let's still better to use the same format.

Reply:
--index-url will replace the default https://pypi.python.org/simple, should use --extra-index-url in addition to default.